src/lib/utils.js

```diff
@@ -211,7 +211,10 @@ export function getDomain(url = '') {
 
 export function formatDate(date, options = { month: 'long', year: 'numeric' }) {
   const d = new Date(date);
-  const locale = typeof window !== 'undefined' ? window.navigator.language : options.locale || 'en-US';
+  const locale =
+    typeof window !== 'undefined'
+      ? window.__NEXT_DATA__.props.pageProps.locale || window.navigator.language
```

Comment:
@ykzts Can we revert that part? I understand that this is an improvement to detect user's locale but I'd rather deprecate the function to force us to use `FormattedDate` instead. This feels a bit hacky and I'm afraid it could break if nextjs change their API.

Reply:
ah, yes. I forgot to revert.

Reply:
[done!](https://github.com/opencollective/opencollective-frontend/compare/9cac806f6f011c0e52e8a8531a9f067fc6c3ebb7..1e79386dc123a302df6b447eada9e71cb03a6e0d)